tests/acceptance/test_internationalization.py

```diff
@@ -69,3 +70,63 @@ def test___supported_encoding___logging_file_path___returns_assigned_value(
     ai_task.in_stream.logging_file_path = file_path  # type: ignore[assignment] # https://github.com/ni/nidaqmx-python/issues/613
 
     assert ai_task.in_stream.logging_file_path == pathlib.Path(file_path)
+
+
+@pytest.mark.grpc_xfail(
+    reason="AB#2393811: DAQmxGetLoggingFilePath returns kErrorNULLPtr (-200604) when called from grpc-device.",
+    raises=DaqError,
+)
+@pytest.mark.parametrize(
+    "file_path, supported_encodings",
+    [
+        ("Zu prüfende Daten.tdms", ["1252", "iso-8859-1", "utf-8"]),
+        ("Données de test.tdms", ["1252", "iso-8859-1", "utf-8"]),
+        ("テストデータ.tdms", ["932", "shift-jis", "utf-8"]),
+        ("테스트 데이터.tdms", ["utf-8", "euc-kr"]),
+        ("测试数据.tdms", ["utf-8", "gbk"]),
+    ],
+)
+def test___supported_encoding___configure_logging___returns_assigned_values(
+    ai_task: Task, file_path: str, supported_encodings: List[str]
+):
+    if _get_encoding(ai_task) not in supported_encodings:
+        pytest.skip("requires compatible encoding")
+    expected_group_name = "Task"
+    expected_logging_mode = LoggingMode.LOG_AND_READ
+    expected_logging_operation = LoggingOperation.CREATE_OR_REPLACE
+
+    ai_task.in_stream.configure_logging(
+        file_path,
+        logging_mode=expected_logging_mode,
+        group_name=expected_group_name,
+        operation=expected_logging_operation,
+    )
+
+    assert ai_task.in_stream.logging_file_path == pathlib.Path(file_path)
+    assert ai_task.in_stream.logging_mode == expected_logging_mode
+    assert ai_task.in_stream.logging_tdms_group_name == expected_group_name
+    assert ai_task.in_stream.logging_tdms_operation == expected_logging_operation
```

Comment:
The work item said to write component tests. I wasn't expecting more i18n acceptance tests, but I don't mind adding them.

Please do the following:
- Copy these tests to tests/component/task/test_in_stream.py and remove the i18n stuff from that copy (use an ASCII filename)
- Remove the non-i18n stuff from this copy (mode, group name, operation)

Reply:
Changed